src/store/modules/global/saga.ts

```
@@ -770,11 +760,40 @@ export function* createBlogSaga(action: PayloadAction<any>): Generator<any, void
     //   yield put({ type: 'GET_ALL_BLOGS_BY_USERID', payload: { id: user.id } })
     // }
   } catch (error: any) {
-    message.error('An unexpected error occurred try again later!')
     console.error('Fetch error:', error)
   }
 }
+export function* createUserBlogSaga(action: PayloadAction<any>): Generator<any, void, any> {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)